src/main/java/com/exasol/dbbuilder/dialects/exasol/VirtualSchema.java

```diff
@@ -235,7 +234,9 @@ public Builder connectionDefinition(final ConnectionDefinition connectionDefinit
          * @return new {@link VirtualSchema} instance
          */
         public VirtualSchema build() {
-            return new VirtualSchema(this);
+            final VirtualSchema virtualSchema = new VirtualSchema(this);
+            this.writer.write(virtualSchema);
+            return virtualSchema;
```

Comment:
Same question

Reply:
A virtual schema has no parent object, no need to check if it was deleted.